### PR TITLE
id field for TimeLog added

### PR DESCRIPTION
**Pull request by Rznjoc**

### CODE REVIEW

File: app/Models/TimeLog.php [hunk 1]
```diff
@@ -11,6 +11,7 @@ class TimeLog extends Model
     use HasFactory;
 
     protected $fillable = [
+        'id',
```
Comment (Prabeg29) [inline app/Models/TimeLog.php:14]: @Rznjoc What is your reason for passing `id` in the $fillable array? Please do check the purpose of this `$fillable` variable.

File: app/Http/Resources/TimeLogResource.php [hunk 2]
```diff
@@ -15,6 +15,7 @@ class TimeLogResource extends JsonResource
     public function toArray($request)
     {
         return [
+            'id'=>$this->id,
             'activity_name'=>$this->activity_name,
             'user_id'=>$this->user_id,
```
Comment (Prabeg29) [inline app/Http/Resources/TimeLogResource.php:20]: @Rznjoc I think, @sabinshrestha8 has done changes for maintaining relationships. So, if I who do not have access to your database, were to use this API via Postman, I would get the following result
` 
  {
     "id": "123",
     "activity_name": "some random activity",
     "user_id": "123",
     "project_id": "123"
      .
      .
      .
  }
`

Now as someone consuming the API, ok I get it this is the 123rd record for TimeLogResource, but I am clueless about who this user is with `id` 123 and the same case for `project_id`.

You might need to consider this change

**Comment by Rznjoc** [comment]
Yea good point dai.. i will work on it.

File: app/Http/Resources/TimeLogResource.php (hunk 2 above)
Comment (Rznjoc) [reply app/Http/Resources/TimeLogResource.php:20]: Understood dai, changes considered.
Screenshot (373)

**Review by Rznjoc — COMMENTED** [review]
id was not needed here in $fillable

**Pull request closed without merging**